optimade/server/entry_collections/mongo.py

```diff
@@ -89,35 +91,21 @@ def count(self, **kwargs) -> int:
             kwargs["filter"] = {}
         return self.collection.count_documents(**kwargs)
 
-    def find(
-        self, params: Union[EntryListingQueryParams, SingleEntryQueryParams]
-    ) -> Tuple[List[EntryResource], int, bool, set]:
-        """Perform the query on the underlying MongoCollection, handling projection
-        and pagination of the output.
+    def insert(self, data: List[EntryResource]):
+        self.collection.insert_many(data)
```

Comment:
This gets used when inserting the test data, but nowhere else

Reply:
Since it becomes an official part of the API for the mongo backend, and can be used to insert other than test data as well, it should at least be documented and perhaps even include some sanity checks. Otherwise, just document it and make it "private".